docs/markdown/guides/testing/bootstrap.init.ts

```diff
@@ -0,0 +1,2 @@
+// Set HANDLE_MERMAID to false to make it already available at test time.
```

Comment:
Is this what you are saying?

```suggestion
// Set HANDLE_MERMAID to false as it's not needed for unit testing
```